ui/src/pages/namespace/Explorer/Tree/index.tsx

```diff
@@ -44,7 +45,23 @@ const ExplorerPage: FC = () => {
   const [deleteNode, setDeleteNode] = useState<BaseFileSchemaType>();
   const [renameFile, setRenameNode] = useState<BaseFileSchemaType>();
   const [previewNode, setPreviewNode] = useState<BaseFileSchemaType>();
+
+  const [search, setSearch] = useState("");
   const { t } = useTranslation();
+  const isSearch = search.length > 0;
```

Comment:
```suggestion
  const hasQuery = query.length > 0;
```
Sometimes the "has" keyword is good for signaling booleans. I think it fits a bit better here than "is".